src/frame/js/actions/conversation.js

```diff
@@ -515,18 +517,31 @@ export function disconnectFaye() {
     };
 }
 
-function handleUserConversationResponse({appUser, conversation, sessionToken}) {
+export function handleUserConversationResponse({appUser, conversation, sessionToken}) {
     return (dispatch) => {
+        Raven.setUserContext({
+            id: appUser._id
+        });
+
         const actions = [
             setUser(appUser),
             setConversation(conversation),
-            setMessages(conversation.messages),
-            setAuth({
-                sessionToken
-            })
+            setMessages(conversation.messages)
         ];
 
+        if (sessionToken) {
+            actions.push(setAuth({
+                sessionToken
+            }));
+        }
+
         dispatch(batchActions(actions));
+
+        if (appUser.conversationStarted) {
```

Comment:
I guess we could, but that'd be a backend logic error if it returns `conversationStarted` for the user and not return the conversation with it.